spyder/plugins/base.py

```diff
@@ -406,6 +406,11 @@ def _refresh_actions(self):
 
         # Create actions list
         self._plugin_actions = self.get_plugin_actions() + additional_actions
+        # Ensure all actions are enabled when adding them to the menu.
+        # See spyder-ide/spyder#11591
+        for action in self._plugin_actions:
+            if action is not MENU_SEPARATOR:
+                action.setEnabled(True)
```

Comment:
Not all actions are enabled or should be enabled when refreshing the Options menu. For example, the Options menu of the editor has one disabled action by default:

![Selección_114](https://user-images.githubusercontent.com/365293/74797147-7171a000-5298-11ea-8ca9-2fac47772c56.png)

How does your proposed solution work in that case?

Reply:
I need to rethink the approach here 🤔 (this is more a workaround than an actual fix of the root cause of the problem to be fair)

Reply:
Although testing a little bit seems like this change doesn't affect the current disabled actions. For example, in the Editor the action `Close this panel` remains disable (I think the plugins and widgets change the state of the actions after the `_refresh_actions` is called):

![help2](https://user-images.githubusercontent.com/16781833/74799702-3c1c8080-529f-11ea-850b-bb686021fb01.gif)

Reply:
> this is more a workaround than an actual fix of the root cause of the problem to be fair

So what could be the cause of this problem?

Reply:
Maybe that an ancestor is being disable or hide when undocking and the actions get disabled. Anyhow, I need to check more deeply what its actually happening here